src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -47,10 +47,16 @@ internal class ExternalXrefMapLoader
             return result;
         }
 
-        public static List<(string, Lazy<ExternalXrefSpec>)> Load(string filePath)
+        public static List<(string, Lazy<ExternalXrefSpec>)> Load(Input input, FilePath filePath)
         {
             var result = new List<(string, Lazy<ExternalXrefSpec>)>();
-            var content = File.ReadAllBytes(filePath);
+            byte[] content;
```

Comment:
this is a special optimization if we can read the file directory.

In line 36, try use call the previous `Load` method if the input is a physical file, and use `JsonUtility.Deserialize` otherwise.

Reply:
agreed